Source/EasyNetQ.Examples.PubSubWithAdvancedApi/EntryPoint.cs

```diff
@@ -2,23 +2,16 @@
 using EasyNetQ.Topology;
 
 var cts = new CancellationTokenSource();
+cts.CancelAfter(300000);
 Console.CancelKeyPress += (_, _) => cts.Cancel();
 
 using var bus = RabbitHutch.CreateBus(
     "host=localhost;publisherConfirms=True",
     x => x.EnableNewtonsoftJson()
         .EnableAlwaysNackWithoutRequeueConsumerErrorStrategy()
-        .EnableConsoleLogger()
```

Comment:
?

Reply:
I have rolled it back. I used these changes to analyse memory allocations.